src/perfectoid_space.lean

```diff
@@ -14,7 +14,7 @@ class perfectoid_ring (R : Type*) extends Tate_ring R :=
 (complete : is_complete R)
 (uniform  : is_uniform R)
 (ramified : ∃ ϖ : units R, (is_pseudo_uniformizer ϖ) ∧ ((ϖ^p : R) ∣ p))
-(Frob     : ∀ a : Rᵒ, ∃ b : Rᵒ, (p : Rᵒ) ∣ (b^p - a))
+(Frob     : ∀ a : Rᵒ, ∃ b : Rᵒ, (p : R) ∣ (b^p - a))
```

Comment:
Why do you make this change? Both `a` and `b` are terms of `Rᵒ`...

Reply:
Because it didn't compile with the other version. I think there have been changes to rings and subrings that broke a lot of small things like like. Kevin should probably check whether he can get this to compile with `Rᵒ`.